cli/pcluster/cli.py

```diff
@@ -181,7 +183,9 @@ def _get_parser():
         default=False,
         help="Disable CloudFormation stack rollback on error.",
     )
-    pupdate.add_argument("-t", "--cluster-template", help="Indicates which section of the cluster template to use.")
+    pupdate.add_argument(
+        "-t", "--cluster-template", help="Indicates the cluster section of the configuration file to use."
```

Comment:
I'd change too: "Indicates which section of the configuration file to use for cluster update."